include/staticjson/basic.hpp

```diff
@@ -23,6 +25,46 @@ struct NonMobile
 
 typedef unsigned int SizeType;
 
+class GlobConfig : private NonMobile
```

Comment:
Rename as "GlobalConfig"?

The shortening is really unnecessary.

Reply:
done. with new commit